docs/application/dotnet/get-started/overview.md

```diff
@@ -22,6 +22,10 @@ When you create your first Tizen .NET application, you can select from the follo
 
     NUI is a rich GUI library used for creating two-dimensional or three-dimensional applications.
     These applications run on a wide range of Tizen devices, such as mobile devices, TVs, and wearables.
+    
+-   [IoT .NET application](iot/first-app.md)
+
+    A IoT application is targeted to run on a IoT board. The applications can be optimized for headed and headless.
```

Comment:
Slight modification needed to correct usage of a or an IoT and making sentences more comprehensible.
```suggestion
An IoT application is targeted to run in an IoT device. The IoT application can be optimized for both the headed as well as the headless IoT devices.

Reply:
Fixed.